src/Tempest/Http/src/Status.php

```diff
@@ -157,7 +159,7 @@ public function description(): string
 
     public function isInformational(): bool
     {
-        return $this->value >= 100 && $this->value < 200;
+        return $this->value < 200;
```

Comment:
What's the advantage of this change?

Reply:
This is the part about phpstan doesn't like sanity checks
> In some condition a "sanity" check was done on the status value, like status >= 100 and status < 600 . But phpstan doesn't like sanity, it likes strict typing ;)

The reason why phpstan errors on this is because it knows $value can be a certain set of values, which start above 100. Meaning `$this->value >= 100`  will always be true. And that is the phpstan error we got.

Reply:
But how would this be correct? `Status::CONTINUE->isInformational()` will now return `true` while it should return `false`?

Reply:
I want to believe you , but... `Status::CONTINUE = 100` correct? meaning the old code `$this->value >= 100 && $this->value < 200`  wil evaluate to `true` . The new code of `$this->value < 200` also returns `true`. If it shouldn't return `true` I can change the expression. But functionally my code is the same as the old one. 

PHPStan noted that because Status starts at `100` the expression `$this->value >= 100` is always true. That is why we can remove it. If we should is another topic. But in my eyes, we would never have a status sub 100, so why not.

Reply:
Ooooh I'm a total fool… I read it as `>=200 and <300` no idea why 😅

Yeah, totally fine this change. Sorry for the noise